app/generator/index.js

```diff
@@ -14,7 +14,7 @@ const generateStatement = async (statement) => {
   const pdfDoc = printer.createPdfKitDocument(docDefinition)
   const { filename, blobUrl } = await publish(pdfDoc, statement, moment(timestamp).format('YYYYMMDDHHmmssSS'))
   await sendPublishMessage(statement, filename)
-  await sendCrmMessage(statement, blobUrl)
+  await sendCrmMessage(statement, blobUrl, filename)
```

Comment:
Do we still need the `blobUrl` or can we remove it now?  If we're publishing this outside our infrastructure for CRM, then just including the one link in the message they need would probably simpler.